components/alert/alert.css

```diff
@@ -0,0 +1,42 @@
+.body {
+    font-family: var(--primary-font);
+}
+
+.para-span {
+
```

Comment:
Remove the extra space i.e. empty new lines.

Reply:
ok removed